src/cable/connection.cr

```diff
@@ -10,7 +10,6 @@ module Cable
     getter token : String?
     getter? connection_rejected : Bool = false
     getter socket
-    getter id : UUID
```

Comment:
I guess we had no use keeping this as a getter, so I just removed it. Every little helps...